### ISSUE TITLE
Find links and filepaths in markdown documents

### ISSUE THREAD

**Issue by lettertwo** [matthewwithanm/markdown-with-front-matter-loader#1]
Should this loader have the ability to generate dependencies from link and source paths, like [html-loader](https://github.com/webpack/html-loader#examples)?  Maybe it could even pass its string content to html loader as a last step?

**Comment by matthewwithanm** [comment]
Yeah, that was the big caveat I gave @Cspeisman when I put this together. I don't think it's quite as simple as passing to html-loader, but I forget why I think that…
